src/FactionMasterInvitationImprove.php

```diff
@@ -60,13 +60,15 @@ public function onLoad(): void {
 		$this->saveResource('en_EN.yml');
 		$this->saveResource('tr_TR.yml');
 		$this->saveResource('pt_BR.yml');
+                $this->saveResource('es_ES.yml');
 		$this->saveResource('ru_RU.yml');
 		$this->saveResource('config.yml');
 		ConfigUpdater::checkUpdate($this, $this->getConfig(), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "fr_FR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "en_EN.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "tr_TR.yml", Config::YAML), "file-version", 1);
 		ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "pt_BR.yml", Config::YAML), "file-version", 1);
+                ConfigUpdater::checkUpdate($this, new Config($this->getDataFolder() . "es_ES.yml", Config::YAML), "file-version", 1);
```

Comment:
To follow the naming of FactionMaster, rename es_ES to es_SPA ?

Reply:
I can rename all the files of FactionMaster to es_SPA if u want

Reply:
because they are not all?

Reply:
In one that you approved it is called es_ES but I have already changed everything and I think I am not forgetting anything.

Reply:
sorry, I didn't pay attention at the time, but it would be more logical :)